src/Streamable.js

```diff
@@ -0,0 +1,8 @@
+import fetch from 'node-fetch';
+
+export const shouldTransform = url => /^https?:\/\/streamable\.com\//.test(url);
```

Comment:
We now have had the following possible links:
- `https://streamable.com/{shortlink}`
- `https://streamable.com/o/{shortlink}` (see `iframe` returned from `oembed.json` call)
- `https://streamable.com/s/{shortlink}`

So I think we should support all of these

If you look at the 3 video's from their homepage (see https://github.com/MichaelDeBoey/gatsby-remark-embedder/pull/48#issuecomment-565169351) and do a right-click to get the embed-code, you also get links that look like:
- `https://streamable.com/s/92ok4/bxmati`
  Is the video same as
  - `https://streamable.com/s/92ok4`
  - `https://streamable.com/92ok4`
- `https://streamable.com/s/clya3/vguntp`
  Is the video same as
  - `https://streamable.com/s/clya3`
  - `https://streamable.com/clya3`
- `https://streamable.com/s/yktcf/jjveaw`
  Is the video same as
  - `https://streamable.com/s/yktcf`
  - `https://streamable.com/yktcf`